src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```diff
@@ -87,6 +95,8 @@ The .NET Foundation licenses this file to you under the MIT license.
 
     <Exec Command="$(IlcHostPackagePath)/native/src/libs/System.Globalization.Native/local_build.sh $(IlcHostPackagePath)/ $(IntermediateOutputPath)" Condition="'$(StaticICULinking)' == 'true'"/>
 
+    <Exec Command="$(IlcHostPackagePath)/native/src/libs/System.Security.Cryptography.Native/local_build.sh $(IlcHostPackagePath)/ $(IntermediateOutputPath)" Condition="'$(StaticOpenSslLinking)' == 'true'"/>
```

Comment:
`local_build.sh` file missing in the PR?

Reply:
Oops. But it's the same as in Globalization work basically.

Reply:
> same as in Globalization

Maybe we should have just one script and pass the library to build as an argument?